.golangci.yml

```diff
@@ -62,11 +61,23 @@ linters-settings:
     local-prefixes: "sigs.k8s.io/cluster-api"
   ginkgolinter:
     # Suppress the wrong length assertion warning.
+    # Default: false
```

Comment:
Is this default true?
Otherwise no need to document the default/explicitly set the var